src/app/frontend/ingress/detail/info.html

```diff
@@ -35,12 +35,13 @@
     </kd-info-card-entry>
   </kd-info-card-section>
   <kd-info-card-section name="[[Rules|Resource rules spec for the ingress.]]">
-    <div ng-repeat="rule in ::$ctrl.ingress.spec.rules">
+    <div ng-if="::$ctrl.ingress.spec.rules.length" ng-repeat="rule in ::$ctrl.ingress.spec.rules">
```

Comment:
This `ng-if` should be used directly at `kd-info-card-section` tag. If there are no rules to display just hide whole section.

Reply:
@floreks I fixed this tag by new commit, thank you for your instruction and advice!!